find_me_a_sandwich/features/step_definitions/favorites_page_steps.rb

```diff
@@ -0,0 +1,53 @@
+Given(/^a user that isn't me has meals saved to favorites$/) do
+  merchant = FactoryGirl.create(:merchant, name: "FOO")
+  menu = FactoryGirl.create(:menu, name: "foobar", merchant_id: merchant.id)
+
+  @other_user = FactoryGirl.create(
+    :user, email: "foo@example.com", password: "testtest"
+  )
+
+  @menu_item_1 = FactoryGirl.create(
+    :menu_item, name: "barbaz A", menu_id: menu.id, menu_group: "foogroup"
+  )
+
+  @menu_item_2 = FactoryGirl.create(
+    :menu_item, name: "barbaz B", menu_id: menu.id, menu_group: "bargroup"
+  )
+
+  FactoryGirl.create(
+    :favorite, user: @other_user, menu_item: @menu_item_1
+  )
+end
+
+When(/^I visit a users populated favorite meals list$/) do
+  visit("/favorites/#{@other_user.id}/")
```

Comment:
As mentioned above, path helpers will go a long way.  You could also follow the Page Objects pattern Ozzy discussed in his tech talk.